budget-api/src/dto/transaction-meta.dto.ts

```diff
@@ -1,5 +1,11 @@
-import { FilterSortPageDto } from './filter-sort-page.dto';
+import { Transaction } from 'src/models/transaction.model';
+import { TransactionFilterSortPageDto } from './transaction-filter-sort-page.dto';
 
-export interface TransactionMetaDto extends FilterSortPageDto {
+export interface TransactionResponse {
```

Comment:
Do you think it could be beneficial to move `TransactionResponse` class into its own file?

Reply:
Yes, this is really good idea